common/prettify.pm

```diff
@@ -1537,6 +1537,7 @@ sub Setup_Handler ($)
         {
             (@{$self->{OUTPUT}})[$self->{FAILED_TESTS_ONLY} ? 0 : 4]->{GIT_CHECKEDOUT_OPENDDS} = $sha;
         }
+        $commits{'commit_OPENDDS'} = $sha;
```

Comment:
How are you testing this? Is there an autobuild variable you're setting in config.xml or something? The GIT_COMMIT environment variable seems to generate a log message, but doesn't update this or impact the junit output.

Reply:
That may be a problem. The code to add those commits is in Prettify::Setup_Handler() which is the same place where the total commit sha entries are added. If a test has those commits, but Junit test page does not show, then it's a problem.

Reply:
So how are you testing this?

Reply:
I added testing notes in the description above.